### ISSUE TITLE
Badges and  Achievemnts 

### ISSUE THREAD

**Issue by mxamusic** [moloch--/RootTheBox#304]
Hi is there a way we can do some badges for challengers, for example, finishing the whole level or 1 category. they unlock achievement or badge

**Comment by jeffg2k** [comment]
Yeah, that would be cool and might fit nicely with the JuiceShop support.

**Comment by mxamusic** [comment]
yeah so we can create multiple badges achievements and even ranks if possible it would, even more, fun if we can have limited time badges so this week badge is python guru and to get this badge u need to solve 1234 challenges which are only available for 24 hrs

**Comment by nunoguerreiro** [comment]
can't wait for this one ... very cool
